FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1066,35 +1105,38 @@ extension ComposeViewController {
 
         present(alert, animated: true, completion: nil)
     }
-}
 
-extension ComposeViewController {
-    private func showScopeAlertIfNeeded() {
-        if shouldRenewToken(for: [.mail]),
-            !userDefaults.bool(forKey: Constants.shouldShowScopeAlertIndex) {
-            userDefaults.set(true, forKey: Constants.shouldShowScopeAlertIndex)
-            let alert = UIAlertController(
-                title: "",
-                message: "compose_enable_search".localized,
-                preferredStyle: .alert
-            )
-            let okAction = UIAlertAction(
-                title: "Log out",
-                style: .default
-            ) { _ in }
-            let cancelAction = UIAlertAction(
-                title: "cancel".localized,
-                style: .destructive
-            ) { _ in }
-            alert.addAction(okAction)
-            alert.addAction(cancelAction)
+    private func showNoAccessToContactsAlertIfNeeded() {
+        guard !cloudContactProvider.isContactsScopeEnabled,
+              !userDefaults.bool(forKey: Constants.didShowContactsScopeAlert)
+        else { return }
 
-            present(alert, animated: true, completion: nil)
+        userDefaults.set(true, forKey: Constants.didShowContactsScopeAlert)
```

Comment:
Do userDefault take into account signing into several accounts in the app? (doesn't look that way)

Reply:
Yes, current implementation doesn't check active user account, I'll fix this

Reply:
It seems currently the only unique key to differentiate accounts is `email`, but it's not very secure to store emails in `UserDefaults`.
Maybe we should just get rid of this alert and show only `Enable Google Contacts Search` button in search results?

Reply:
That would be easiest, and adds consistency across platforms